website/docs/api/soql.md

```diff
@@ -947,25 +947,6 @@ SOQL.of(Account.sObjectType)
 SOQL.setMock('MyQuery', 5);
 ```
 
-### static resource mock
-
-**Signature**
-
-```apex
-SOQL setMock(String mockId, String staticResource)
-```
-
-**Example**
-
-```apex
-SOQL.of(Account.sObjectType)
-    .mockId('MyQuery')
-    .toList();
-
-// In Unit Test
-SOQL.setMock('MyQuery', 'MyStaticResource');
```

Comment:
Maybe leave just that one line with:
```java
SOQL.setMock('MyQuery', Test.loadData(SObject.SObjectType, 'StaticResourceName'));
```
It will be helpful to let know that this is possible.